Makefile

```diff
@@ -0,0 +1,10 @@
+identity={"identity": {"org_id": "3340851", "type": "System", "auth_type": "cert-auth", "system": {"cn": "1b36b20f-7fa0-4454-a6d2-008294e06378", "cert_type": "system"}, "internal": {"org_id": "3340851", "auth_time": 6300}}}
+b64_identity=$(shell echo '${identity}' | base64 -w 0 -)
+
+file=./scripts/samples/cost-mgmt.tar.gz
+
+local-upload-data:
+	curl -vvvv -F "upload=@$(file);type=application/application/vnd.redhat.hccm.tar+tgz" \
+		-H "x-rh-identity: ${b64_identity}" \
+		-H "x-rh-request_id: testtesttest" \
+		http://localhost:3000/api/ingress/v1/upload
```

Comment:
Can we also get the port from the ENV var?

Reply:
Yes, we can but is there any use case for it? We have hard coded ingress port to 3000 in docker-compose, if we add env variable then we will also need to change docker-compose.yaml.

Reply:
The use case is same, I have another services running on the same port and instead of editing code to run the setup I can export env var?

Reply:
Done, I have made ingress port configurable.